src/main/java/br/com/labbs/monitor/filter/MetricsCollectorFilter.java

```diff
@@ -209,20 +222,36 @@ private boolean isErrorStatus(int status) {
     /**
      * Get the error message from a request.
      * If error message is null, sets the string to empty string.
+     * If a regex is defined, use it to filter message
+     * 
+     * Default regex: ^([a-zA-z0-9 ]{0,120})
+     * Default max size: 120
      *
      * @param httpRequest request
      * @return string with the error message or empty string if error message not found.
      */
     private String getErrorMessage(HttpServletRequest httpRequest) {
-    	if (errorMessageParam == null) {
-    		return "";
-    	}
-    	String errorMessage = (String) httpRequest.getAttribute(errorMessageParam);
-    	if (errorMessage == null) {
-    		return "";
-    	}
 
-    	return errorMessage;
+        if (errorMessageParam == null) {
+            return "";
+        }
+        String errorMessage = (String) httpRequest.getAttribute(errorMessageParam);
+        String result = "";
+        if (errorMessage == null) {
+            return result;
+        }
+
+        try {
+            // apply whitelist filter
+            result = errorMessage.replaceAll(filter_regex, "");
+            if (result.length() > filter_max_size) {
+                result = result.substring(0, filter_max_size);
+            }
+        } catch (Exception e) {
+            // avoid invalid regex or invalid matcher group index
```

Comment:
Log here the exception with "warning" for the app developer to know that something is wrong.